src/sections/DonateSectionOne/DonateSectionOne.scss

```
@@ -0,0 +1,57 @@
+@import 'styles/utilities/breakpoints.scss';
+
+.donate-section-one {
+  max-width: 900px;
+  margin-top: 60px;
+  margin-bottom: 96px;
+
+  @media (max-width: $breakpoint-tablet) {
+    margin: 0 30px;
+  }
+
+  &__title {
+    margin-bottom: 32px;
+
+    @media (max-width: $breakpoint-mobile) {
+      text-align: center;
+    }
+  }
+
+  &__body {
+    display: flex;
+    justify-content: space-between;
+
+    @media (max-width: $breakpoint-tablet-large) {
+      flex-direction: column-reverse;
+    }
+  }
+
+  &__content {
+    @media (max-width: $breakpoint-mobile) {
+      text-align: center;
+    }
+  }
+
+  &__text {
+    margin-bottom: 32px;
+    @media (max-width: $breakpoint-mobile) {
+      text-align: left;
+    }
+  }
+
+  &__image {
+    width: 280px;
+    height: 280px;
+    margin-left: 36px;
+
```

you should have object-fit in here so the image adjusts to its container.

```suggestion
    margin-left: 36px;
    object-fit: cover;
```

should do the trick